src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -337,13 +337,13 @@ SELECT find_shard_interval_index(1300016);
 (1 row)
 
 -- check external colocation API
-SELECT count(*) FROM pg_dist_partition WHERE colocationid = 4;
+SELECT count(*) FROM pg_dist_partition WHERE colocationid IN (4, 5);
```

Comment:
minor: why do we need to add `5` here?

Reply:
There was a fail saying there is another partition with the id 5.

Reply:
I couldn't follow. We only have `WHERE = 4` in the query, how does not adding `5` would change the query result?

Reply:
Sorry, I meant the fail says, there is another colocation with id 5. This just checks if there is a partition with the colocation id 4. And the next line deletes the colocation from `pg_dist_colocation`. So I did the same thing for id 5 too.